src/test/java/com/wikia/webdriver/common/remote/operations/http/BaseRemoteOperation.java

```diff
@@ -50,7 +50,7 @@ String execute(final HttpEntityEnclosingRequestBase request, final JSONObject js
       request.setHeader(HttpHeaders.CONTENT_TYPE, ContentType.APPLICATION_JSON.toString());
       request.setEntity(new StringEntity(jsonObject.toString()));
       result = execute(request);
-    } catch (UnsupportedEncodingException x) {
+    } catch (UnsupportedEncodingException|RemoteException x) {
```

Comment:
Nitpick: you're missing spaces before and after the pipe.
I'd suggest naming the exception variables `ex`.